src/Calculator/Views/DateCalculator.xaml

```diff
@@ -417,35 +417,23 @@
             </Grid.RowDefinitions>
 
             <!-- From Date -->
-            <TextBlock x:Name="Date_FromLabel"
-                       x:Uid="Date_FromLabel"
-                       Grid.Row="1"
```

Comment:
Since we are removing some rows, we should update `Grid.RowDefinitions` to remove those rows as well. That also means we should update `Grid.Row` of the controls we are not removing to make sure they align with the new row definitions (e.g., the first CalendarDatePicker would now be `Grid.Row="1"` instead of `Grid.Row="2"`

Reply:
I remapped the grid components as per your instruction but components seem to appear to be squished together which i think is due to the absence of the extra padding that was present because of the text block before the calendar picker/text block consolidation.
 
Tried adding extra padding(space occupied by the removed textblocks) and the components seem to be nicely spaced out as before.So should i do this or is there another alternative?

Reply:
Hmm, I see what you mean. Instead of removing rows 0 and 3 (the rows that were adding that padding), try removing rows 1 and 4 (the rows that those removed TextBlocks were in). That _should_ preserve that spacing, but if not, I can take a closer look and suggest an alternative.

Reply:
That seemed to do the trick.